src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -613,5 +614,55 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
     } else if (topinventory.getTitle == LIGHT_PURPLE.toString + "" + BOLD + "スキルを進化させました") {
       event.setCancelled(true)
     }
+
+  }
+
+
+  @EventHandler
+  def onItemNameRemoveEvent(event: InventoryCloseEvent): Unit = {
+    val player = event.getPlayer.asInstanceOf[Player]
+  //エラー分岐
+    val inventory = event.getInventory
+
+    //インベントリサイズが36でなければ処理を終了させる
+    if (inventory.row != 4){
+      return
+    }
+    if (inventory.getTitle == s"${GOLD}${BOLD}名義をなくしたいアイテムを投入してください") {
+      val item = inventory.getContents
+
+      var count = 0
+      //for文を使い、1つずつアイテムを見ていく
+      for(m <- item){
+        if(m !=null){
+          if (m.getItemMeta.hasLore){
```

Comment:
`ItemStack`が`ItemMeta`を持っているかどうかも確認する必要があります。
```suggestion
          if (m.hasItemMeta && m.getItemMeta.hasLore) {
```

Reply:
これ治ってないっぽいですよ

Reply:
`item.getItemMeta.hasLore`を2回確認しているようです

Reply:
あれ… 本当ですね